src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,53 @@
+import {useEffect, useState} from "react";
+import {useParams, Link} from "react-router-dom";
+import {getArticleById} from "../utils/api";
+import VoteCard from "./VoteCard";
+
+export default function SingleArticle() {
+  const {article_id} = useParams();
+  const [isLoading, setIsLoading] = useState(false);
+  const [article, setArticle] = useState([]);
```

Comment:
It's good practice to use the same data type for the default as what you expect it to be - so article will be an object, so your initial value would be an empty object.